bin/inx-chronicle/src/launcher.rs

```diff
@@ -32,6 +32,8 @@ pub struct LauncherState {
     config: ChronicleConfig,
     #[allow(dead_code)]
     db: MongoDb,
+    #[cfg(feature = "api")]
+    secret_key: libp2p_core::identity::ed25519::SecretKey,
```

Comment:
Do we really need an `ed25519` secret key, or could we go with something more common? Maybe Bee just uses `ed25519` because the node requires it anyways?

Reply:
We can use pretty much anything. What do you suggest?

Reply:
Ok, I did some more digging and `ed25519` seems to be a good choice, sorry about that. We should, however, use a more basic library, such as `ed25519-dalek` instead of `libp2p_core` (which has bitten us from time to time).